packages/code-du-travail-frontend/sentry.client.config.js

```diff
@@ -10,4 +10,8 @@ Sentry.init({
   environment:
     process.env.NEXT_PUBLIC_SENTRY_ENV || process.env.SENTRY_ENV || "dev",
   tracesSampleRate: 1.0,
+  release: process.env.NEXT_PUBLIC_SENTRY_RELEASE || process.env.SENTRY_RELEASE,
+  replaysSessionSampleRate: 1.0,
+  replaysOnErrorSampleRate: 1.0,
+  integrations: [new Sentry.Replay()],
```

Comment:
permet de suivre le parcours avant l'erreur : https://sentry.io/for/session-replay/  (c'est [gdpr friendly](https://docs.sentry.io/product/session-replay/protecting-user-privacy/))

Reply:
D'ailleurs il faudrait réduire le `replaySessionSampleRate` pour que ça recorde que 5 ou 10% max des sessions où il y a pas d'erreur

Reply:
⚠️ ce feature ne fonctionne pas, j'ai pas réussi à comprendre la raison, car tout est bien envoyé, et on dispose égalmeent d'un `relayId` sur les issues, mais rien s'affiche